dashboard2docker.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 cp grafana-dashboards/TeslaMetricsV2.json provisioning/dashboards/TeslaMetricsV2.json
-sed -i 's/${DS_TESLA}/Tesla/g' provisioning/dashboards/*.json
+sed -i.bak 's/${DS_TESLA}/Tesla/g' provisioning/dashboards/*.json
```

Comment:
Works on Mac & Linux (OSX sed is unfortunately not GNU-compatible, it's from BSD)